cranelift/filetests/filetests/isa/aarch64/condops.clif

```diff
@@ -763,14 +763,14 @@ block0(v0: i8, v1: i8, v2: i8):
 ;   orr w0, w5, w7
 ;   ret
 
-function %i(b1, i8, i8) -> i8 {
-block0(v0: b1, v1: i8, v2: i8):
+function %i(i8, i8, i8) -> i8 {
+block0(v0: i8, v1: i8, v2: i8):
   v3 = select.i8 v0, v1, v2
   return v3
 }
 
 ; block0:
-;   and w5, w0, #1
+;   uxtb w5, w0
```

Comment:
I think this could get a bit better (one instruction shorter) with

```
tst w0, #255
csel ...
```

(`tst w0, #255` is an alias for `ands wzr, w0, #255`; if we don't have machinst support for that yet it should be easy as another ALU op, one bit different from `and`)

Reply:
Would you be okay with addressing this one in follow-up PR?

Reply:
Yep, that's fine!

Reply:
I went ahead and fixed it on this branch, and found a bug with the x64 lowering of `select` in the process!